### ISSUE TITLE
Add the ability to auto assign a PR to a contributor through reviewable

### ISSUE THREAD

**Issue by TravisEz13** [Reviewable/Reviewable#288]
Add the ability to auto-assign a PR to a contributor through reviewable.  We are having to teach everyone how to go through and find the reviews they are responsible for.  It would be much easier if we could just assign the reviews.

**Comment by pkaminski** [comment]
Can you explain your use case in more detail, please?  If you want to set an assignee when you first create a PR, can you do it on the same page in GitHub?  Or do you want some kind of autonomous system to determine the "best" assignee for a PR?

**Comment by TravisEz13** [comment]
We have assigned contributors who review certain repo (for smaller projects.)  most of our external PR requests, don't know, and from what they say, cannot assign the PRs.

**Comment by pkaminski** [comment]
OK, that makes sense.  How about if I extended the [review completion rule](https://github.com/Reviewable/Reviewable/wiki/FAQ#can-i-customize-what-makes-a-review-complete) SPI to also be able to return a list of assignees?  Would that be sufficient?

(This might also help @pcorpet with #287.)

**Comment by TravisEz13** [comment]
Yes, I think this would work (I looked to see if this is how it worked already.)  If it doesn't, it's my org's fault.
